joomla/joomla-cms
# Convert the content plugins to service providers

### Summary of Changes
Converts the rest of the content plugins to service providers.

### Testing Instructions
- Enable all content plugins
- Load a module in an article
- Add a page break to an article
- Save the article
- Check if the article is indexed
- Check if the module is loaded on the front end

### Actual result BEFORE applying this Pull Request
All works.

### Expected result AFTER applying this Pull Request
All works.

### Link to documentations
Please select:
